src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/Mocks/JwtTokenMock.cs

```diff
@@ -35,41 +35,9 @@ public static string GenerateToken(ClaimsPrincipal principal, TimeSpan tokenExip
             return tokenstring;
         }
 
-        /// <summary>
-        /// Validates a token and return the ClaimsPrincipal if successful. The validation key used is from the self signed certificate
-        /// and is included in the integration test project as a separate file.
-        /// </summary>
-        /// <param name="token">The token to be validated.</param>
-        /// <returns>ClaimsPrincipal</returns>
-        public static ClaimsPrincipal ValidateToken(string token)
-        {
-            string unitTestFolder = Path.GetDirectoryName(new Uri(typeof(JwtTokenMock).Assembly.CodeBase).LocalPath);
-
-            string certPath = Path.Combine(unitTestFolder, @"..\..\..\JWTValidationCert.cer");
-
-            X509Certificate2 cert = new X509Certificate2(certPath);
-            SecurityKey key = new X509SecurityKey(cert);
-
-            TokenValidationParameters validationParameters = new TokenValidationParameters
-            {
-                ValidateIssuerSigningKey = true,
-                IssuerSigningKey = key,
-                ValidateIssuer = false,
-                ValidateAudience = false,
-                RequireExpirationTime = true,
-                ValidateLifetime = true
-            };
-
-            JwtSecurityTokenHandler validator = new JwtSecurityTokenHandler();
-            return validator.ValidateToken(token, validationParameters, out SecurityToken validatedToken);
-        }
-
         private static SigningCredentials GetSigningCredentials()
         {
-            string unitTestFolder = Path.GetDirectoryName(new Uri(typeof(JwtTokenMock).Assembly.CodeBase).LocalPath);
-
-            string certPath = Path.Combine(unitTestFolder, @"..\..\..\jwtselfsignedcert.pfx");
-            X509Certificate2 cert = new X509Certificate2(certPath, "qwer1234");
+            X509Certificate2 cert = new X509Certificate2("selfSignedTestCertificate.pfx", "qwer1234");
```

Comment:
is the content of selfsignedTestcert.pfx the same a for jwtselfsigned? or should we change to using this certificate in other test projects as well ?

Reply:
It is the same certificate. I renamed it for the test project here to make extra sure that there are no resource use between the projects.